src/main/java/stobberi/components/Parser.java

```diff
@@ -0,0 +1,82 @@
+package stobberi.components;
```

Comment:
Package name is in lower cases, class names are in PascalCase and variable names are in camelCase. Great job following the naming convention throughout!

Reply:
Thank you!